python/options.py

```diff
@@ -0,0 +1,55 @@
+import argparse
+
+parser = argparse.ArgumentParser()
+
+parser.add_argument('file',type=str, nargs='+')
+parser.add_argument('--OutputName',action="store",dest="OutputName",type=str,default="Myconfig",help="Set Output file name, default is: Myconfig")
+parser.add_argument('--SelectSheetNum',action="store",dest="SelectSheetNum",type=int,default=0,help="Select the Excel sheet for reading, default is: 0")
+parser.add_argument('--SelectColumnX',action="store",dest="SelectColumnX",type=int,default=0,help= "Select Column number for X data, default value is 0")
+parser.add_argument('--SelectColumnY',action="store",dest="SelectColumnY",type=int,default=0,help= "Select Column number for Y data, default value is 0")
+parser.add_argument('--SelectRowStart',action="store",dest="SelectRowStart",type=int,default=1,help= "Select the first Row for reading, default is 1")
+parser.add_argument('--SelectRowEnd',action="store",dest="SelectRowEnd",type=int,default=61,help= "Select the last Row for reading, default is 61")
+parser.add_argument('--AxisNDiv',action="store",dest="AxisNDiv",type=str,default="508,510",help="Set Axis Ndivisions X,Y (default is: 508,510)")
+parser.add_argument('--CanvDim',action="store",dest="CanvDim",type=str,default="1000,1000",help="Set Canvas Dimensions X,Y (default is: 1000,1000)")
+parser.add_argument('--CanvDrawOpt',action="store",dest="CanvDrawOpt",type=str,default="APE1",help="Set Draw Option (default is: APE1)")
+parser.add_argument('--CanvGridXY',action="store",dest="CanvGridXY",type=str,default="false,false",help="Set GridXY option (default: false,false)")
+parser.add_argument('--LatexLine',action="store",dest="LatexLine",type=str,default="0.62,0.86, #splitline{LS2}{Detector~Production}",help= "Add Latex line, example: 0.62,0.86, #splitline{LS2}{Detector~Production}")
+parser.add_argument('--LatexLine1',action="store",dest="LatexLine1",type=str,default="0.62,0.27, Gas~=~CO_{2}",help= "Add Latex line, example: 0.62,0.86, #splitline{LS2}{Detector~Production}")
+parser.add_argument('--LatexLine2',action="store",dest="LatexLine2",type=str,default="",help= "Add Latex line, example: 0.62,0.86, #splitline{LS2}{Detector~Production}")
+parser.add_argument('--LatexLine3',action="store",dest="LatexLine3",type=str,default="",help= "Add Latex line, example: 0.62,0.86, #splitline{LS2}{Detector~Production}")
+parser.add_argument('--LatexLine4',action="store",dest="LatexLine4",type=str,default="",help= "Add Latex line, example: 0.62,0.86, #splitline{LS2}{Detector~Production}")
+parser.add_argument('--LatexLine5',action="store",dest="LatexLine5",type=str,default="",help= "Add Latex line, example: 0.62,0.86, #splitline{LS2}{Detector~Production}")
```

Comment:
Lines 16-21; This is not a good implementation.  It's tedious and limits the user to only allowing a maximum of 6 lines.  I think it would be better if you had something like this using the `append` action:

```
parser. add_option('--latexLines', action="append", type="string", dest="latexLines", default=None, help= "Add an entry to the latexLines list following syntax: 'Coord_PadX,Coord_PadY, text'", metavar="latexLines")
```

See documentation: https://docs.python.org/2/library/optparse.html#other-actions